### ISSUE TITLE
Release v1.31.10

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#1261]
:robot: A dedicated issue to run tests and deploy release [v1.31.10](https://github.com/deckhouse/deckhouse/milestone/40).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | gcp | azure | openstack | yandex-cloud | vsphere | static`
  - `git_ref` is a tag or branch: `vX.Y.Z | test-X.Y.Z* | main | release-X.Y`
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `containerd | docker`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.19 | 1.20 | 1.21 | 1.22`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag or branch: `vX.Y.Z | test-vX.Y.Z* | main | release-X.Y`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.19
/e2e/use/k8s/1.21

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.19 and 1.21.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build modules FE' and 'build FE' jobs are finished.

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [07d045](https://github.com/deckhouse/deckhouse/commit/07d045ac4dee497a1ae2dfaa256c9d1cb4f700fe) in branch **main**:
```
[cloud-provider-openstack] Remove default podNetworkMode from config-values (#1248)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ea5ef5](https://github.com/deckhouse/deckhouse/commit/ea5ef50bc476453bc68afd34a81081a692aabbf7) in branch **main**:
```
[ingress-nginx] VHost Detail: select all locations by default (#1251)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [db310e](https://github.com/deckhouse/deckhouse/commit/db310e8f25f469f77ce20ded951cf17882d2e12c) in branch **main**:
```
Re-generate changelog (#1262)

Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.31.10** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [6517b5](https://github.com/deckhouse/deckhouse/commit/6517b5ec87ea7d827caf7573ad86ca7406800d23) in branch **main**:
```
[user-authn] Dex usage doc fix due to new default Gitlab settings (#1236)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [8527bd](https://github.com/deckhouse/deckhouse/commit/8527bd90b654ccc16314d440cf9fda349f2ddd4d) in branch **main**:
```
[cloud-provider-aws] allow ssh access to master nodes only from bastion for with-nat layout (#1217)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b088fa](https://github.com/deckhouse/deckhouse/commit/b088fa456683fc7f45e33c4383dbd03c4aae8399) in branch **main**:
```
[Upmeter] Fix the correctness of group stats (#1144)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [687519](https://github.com/deckhouse/deckhouse/commit/687519803785ebf3c5e50982d043965670cabd4d) in branch **main**:
```
[cloud-provider-aws::docs] add policies for creating peering connection (#504)
```
